### PR TITLE
kawaii-player: Add Version 5.0.0-1

### PR DESCRIPTION
**Pull request by SonaliBendre**
Closes #XXXX

Relates to #XXXX

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/kawaii-player.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    "homepage": "https://github.com/kanishka-linux/kawaii-player",
+    "license": "GPL-3.0-only",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/kanishka-linux/kawaii-player/releases/download/v5.0.0-1/kawaii-player-5.0.0-1.windows10.x86_64-bin.7z",
+            "hash": "7249072cdd4914a0da276fe1c2990579df486555dce5dcc6cd6f889c22e90864",
+            "extract_dir": "kawaii-player-5.0.0-1.windows10.x86_64-bin"
+        }
+    },
+    "bin": "kawaii_player.exe",
```
Comment (issaclin32) [inline bucket/kawaii-player.json:13]: Does the app support command-line **arguments**. We only add `bin` for GUI apps if they support command-line args.

**Comment by issaclin32** [comment]
Please change the **title** of your PRs to `(PACKAGE NAME): Add version xxx.`

File: bucket/kawaii-player.json (hunk 1 above)
Comment (SonaliBendre) [reply bucket/kawaii-player.json:13]: There's not any arguements but we can launch it using terminal

Comment (issaclin32) [reply bucket/kawaii-player.json:13]: > There's not any arguements but we can launch it using terminal

That doesn't match the criteria of using `bin`. Please remove `bin`.

**Comment by issaclin32** [comment]
config location: `$UserProfile\.config\kawaii-player`
(This is a reminder. No need to modify anything regard to this.)

**Pull request merged**